### ISSUE TITLE
OBJ metallic/roughness material texture references shown only if a factor is present as well

### ISSUE THREAD

**Issue by mosra** [ufbx/ufbx#55]
I was creating some minimal test files by hand and came across the following -- if an OBJ `*.mtl` file has this content, the imported material doesn't contain any texture references:

```obj
newmtl Material
map_Pm a.png
map_Pr b.png
```

While, if I do the following, it does. It does include the textures also if I set metalness to `0.0`:

```obj
newmtl Material
Pm 1.0
map_Pm a.png
map_Pr b.png
```

For e.g. `Kd` this works well, `map_Kd` is recognized even without `Kd` being present. 

Could it be perhaps related to https://github.com/ufbx/ufbx/pull/50? I.e., the code should be checking for the `map_*` values as well? Or, probably best to extend the check to all PBR properties listed at http://exocortex.com/blog/extending_wavefront_mtl_to_support_pbr.

Thank you! (And sorry for neglecting PR reviews these days, I'll get to it as soon as I can.)

**Comment by bqqbarbhg** [comment]
Thanks for the clear repro case! The textures were recognized without issue but the Magnum importer didn't consider the fields as the `ufbx_material_features.pbr` wasn't enabled. Your intuition about it being related to #50 is correct, `UFBXI_SHADER_FEATURE_IF_EXISTS` used for implementing that didn't consider textures.

Got a fix implemented in #57, and no sweat with the PR review, I'm not in a rush waiting for it (and your reviews are in-depth enough that they probably aren't done in a minute :D)

**Comment by bqqbarbhg** [comment]
Should be fixed as of [v0.3.0](https://github.com/ufbx/ufbx/tree/v0.3.0), leaving the issue open in case you want to verify on your side, free free to close otherwise!

**Comment by mosra** [comment]
Everything works great now, thanks a lot for the ultra-fast fix! :+1:
